packages/console/src/containers/AppContent/components/Topbar/TenantSelector/index.module.scss

```diff
@@ -0,0 +1,116 @@
+@use '@/scss/underscore' as _;
+
+.currentTenantCard {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: _.unit(1);
+  max-width: 500px;
```

Comment:
nit: fall all the max-width, better give a scss variable, in case the max width setting is shared by some other components.